manager/orchestrator/replicated/services.go

```diff
@@ -50,7 +50,7 @@ func (r *Orchestrator) handleServiceEvent(ctx context.Context, event events.Even
 		if !orchestrator.IsReplicatedService(v.Service) {
 			return
 		}
-		orchestrator.DeleteServiceTasks(ctx, r.store, v.Service)
+		orchestrator.SetServiceTasksRemove(ctx, r.store, v.Service)
```

Comment:
You need to make the same change in `github.com/swarmkit/manager/orchestrator/global/global.go` `L150`.

Then, you can remove the `DeleteServiceTasks` method, because it has no other uses in the codebase.

Reply:
Done.